pyproject.toml

```diff
@@ -31,6 +31,7 @@ psycopg2-binary = { version = "^2.9.9", optional = true }
 pymysql = { version = "^1.1.0", optional = true }
 alembic = "^1.13.1"
 alembic-postgresql-enum = "^1.2.0"
+numpy = "<2.0.0"
```

Comment:
This is an optional dependency and enable only with `visualization` extras. `numpy` is used by `matplotlib` but when installed, `networkx` will use it and need a version `<2.0.0`.